packages/hash/playwright/tests/guest-user.spec.ts

```diff
@@ -2,7 +2,7 @@ import { test, expect } from "@playwright/test";
 
 test("guest user navigation to login and signup pages", async ({ page }) => {
   await page.goto("/");
-  await page.waitForNavigation({ url: "**/login" });
+  await page.waitForURL("**/login");
```

Comment:
`waitForNavigation` failed for me in CI, so I investigated it and found the fix. Unlike `waitForNavigation`, `waitForURL` succeeds even if we are on the right page already. Not sure if I have fixed some general test flakiness by this or this was a new glitch. It might be related to a change in `_app.page.tsx`.